.github/workflows/test.yml

```diff
@@ -47,5 +47,4 @@ jobs:
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: us-east-1
       - run: |
-          sleep 10
```

Comment:
@dim still relevant? (I'd say - don't sleep, but don't want to experiment with this)